sdcm/tester.py

```diff
@@ -1672,3 +1672,14 @@ def list2reason(exc_list):
             TEST_LOG.error("-" * 70)
             TEST_LOG.error(text)
             TEST_LOG.error("-" * 70)
+
+        self.tag_ami_with_result(error, failure)
+
+    def tag_ami_with_result(self, test_error, test_failure):
+        if self.params('cluster_backend', '') == 'aws' and self.params('tag_ami_with_result'):
+            test_result = 'PASSED' if not test_error and not test_failure else 'ERROR' if test_error else 'FAILURE'
```

Comment:
this is too long and unreadable, convert to regular if

Reply:
fixed